0020-day-two/stand-ups/readme.md

```diff
@@ -0,0 +1,99 @@
+
+## Facilitation Guidance
+
+The scrum master should facilitate the stand-up meeting.
+
+### Answer 3 Simple Questions
+
+Each contributor should only answer the following 3 questions in 1 sentence or less:
+
+### Speak to Tasks
+
+When answering what was worked on and what will be worked on, refer directly to tasks. This has two benefits
+
+1. The answer will naturally be short (e.g. "I finished task 114 yesterday; which was to update the build of the api container image. I will be starting on task 115 today; which is to update the release pipeline for the same container image.").
+2. Unplanned work will be easily identifiable. If the person is unable to refer to a task, that typically indicates they are working on something unplanned or out of scope for the sprint (e.g. Yesterday, I was attempting to optimize some of the unit tests from last sprint to run faster).
+
+If a contributor is not working on an existing sprint task they need to either create a new task under an existing sprint story to reflect that work, or defer that work until its scheduled for a later sprint.
+
+**If a contributor provides an update without referring to a task, ask the contributor which task.**
+
+### Parking Lot Discussion Items
+
+As contributors are answering the three questions, if another contributor has a question or issue to share, they should reserve until after all contributors have answered finished. Once each member has answered all three questions, the scrum master should open up the floor to anyone who may have an open question or unresolved issue to share. This portion of the ceremony is often referred to as the "Parking Lot".
+
+**Parking lot discussions are optional for participants**. Ensure discussion leaders call out necessary parties for their discussion points upfront, allowing those not needed to leave the meeting.
+
+### Start On Time
+
+Make a best effort to begin answering the 3 questions as close to the scheduled start time as possible. Try not to waste time upfront on chit-chat or waiting on all team members to join. This can extend the meeting time significantly. Starting immediately will help ensure stand-ups remain effective and useful over time.
+
+### Same Time Everyday
+
+Stand-up should be held at the same time each day. The team will know each day at 10am they have stand-up and begin to plan their day around this daily ceremony.
+
+### Schedule Stand-ups for Mid-Morning
+
+The ideal time for stand-ups is between 9 and 10am. This provides people with ample time to get into the office, settle in, and catch up with where they left off the day before. Having stand-ups at this time will help minimize distractions such as getting coffee, checking emails, etc.

```

Comment:
See comment above.

Reply:
IMO this is too prescriptive anyways. I'm going to change the language a bit.

Reply:
Fixed in https://github.com/microsoft/code-with-engineering-playbook/pull/67/commits/5640c9974b706f23129563781895556ae63f2be9.